src/main/java/com/plugin/HttpOutput.java

```diff
@@ -20,37 +28,85 @@
 import com.google.inject.Inject;
 import com.google.inject.assistedinject.Assisted;
 
-import okhttp3.MediaType;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.RequestBody;
-import okhttp3.Response;
-
 /**
  * This is the plugin. Your class should implement one of the existing plugin
  * interfaces. (i.e. AlarmCallback, MessageInput, MessageOutput)
  */
 public class HttpOutput implements MessageOutput {
 
+    public static final int HTTP_BATCH_SIZE = 500;
```

Comment:
please use the same indentation everywhere (rest of the code seems to use 2 spaces)